website/docs/cli/configuration/configuration.mdx

```diff
@@ -678,6 +678,13 @@ setting `ATMOS_STACKS_BASE_PATH` to a path in `/localhost` to your local develop
 | ATMOS_SETTINGS_LIST_MERGE_STRATEGY                    | settings.list_merge_strategy                    | Specifies how lists are merged in Atmos stack manifests. The following strategies are supported: `replace`, `append`, `merge`                                                                                                |
 | ATMOS_VERSION_CHECK_ENABLED                           | version.check.enabled                           | Enable/disable Atmos version checks for updates to the newest release                                                                                                                                                        |
 | ATMOS_GITHUB_TOKEN                                    | N/A                                             | Bearer token for GitHub API requests, enabling authentication for private repositories and higher rate limits                                                                                                                |
+| ATMOS_BITBUCKET_TOKEN                                 | N/A                                             | Bitbucket app password for API requests; used to avoid rate limits. Unauthenticated requests are limited to 60 requests per hour. When both ATMOS_BITBUCKET_TOKEN and BITBUCKET_TOKEN are defined, the former prevalis       |
```

Comment:
```suggestion
| ATMOS_BITBUCKET_TOKEN                                 | N/A                                             | Bitbucket App Password for API requests; used to avoid rate limits. Unauthenticated requests are limited to 60 requests per hour. When both ATMOS_BITBUCKET_TOKEN and BITBUCKET_TOKEN are defined, the former prevails       |
```